.github/workflows/go.yml

```diff
@@ -134,7 +134,7 @@ jobs:
 
   windows:
     name: AMD64 Windows 2019 Go ${{ matrix.go }}
-    runs-on: windows-latest
+    runs-on: windows-2019
```

Comment:
@emkornfield Does go care if running on Windows 2019 or 2022?

Reply:
@zeroshade is in the best position to answer.